test/clojure_tictactoe/cli/output/board_printer_test.clj

```diff
@@ -0,0 +1,24 @@
+(ns clojure-tictactoe.cli.output.board-printer-test
+  (:require [clojure.test :refer :all]
+            [clojure.string :as string]
+            [clojure-tictactoe.helpers :as helper]
+            [clojure-tictactoe.cli.output.board-printer :refer :all]))
+
+(testing "Rendering board"
+  (testing "render-board"
+    (deftest render-board-test
+      (testing "Displays a simple blank board"
+        (is (= " 0 | 1 | 2 \n===+===+===\n 3 | 4 | 5 \n===+===+===\n 6 | 7 | 8 \n" (render-board [0 1 2 3 4 5 6 7 8]))))))
+
+  (testing "render-board-spaces"
+    (deftest render-board-spaces-with-no-arguments-test
+      (testing "render-board-spaces with no arguments returns spaces that haven't been marked"
+        (is (= (render-board-spaces) [0 1 2 3 4 5 6 7 8]))))
+
+  (deftest render-board-spaces-with-a-move-on-space-0-test
+    (testing "Returns spaces where only the first space has been marked"
+      (is (= (render-board-spaces ["x" 1 2 3 4 5 6 7 8]) ["x" 1 2 3 4 5 6 7 8]))))
+
+  (deftest render-board-spaces-with-a-move-on-multiple-spaces-test
+    (testing "Returns spaces where the appropriate space is marked"
+      (is (= (render-board-spaces [0 1 "x" "x" 4 5 6 "o" 8]) [0 1 "x" "x" 4 5 6 "o" 8]))))))
```

Comment:
Do you still need `render-board-spaces` if it's just going to return the argument that it's given?

Reply:
I do not!